estate-services/src/test/java/org/egov/ps/test/validator/PropertyValidatorServiceTest.java

```diff
@@ -110,7 +110,7 @@ public void testvalidateDocumentsOnTypeNegative() throws Exception {
 
 		// Step 4 - Test case
 		Map<String, String> errorMap = new HashMap<String, String>();
-		propertyValidator.validateDocumentsOnType(requestInfo, "ch", owner, errorMap, "");
+		propertyValidator.validateDocumentsOnType(requestInfo,"ESTATE_BRANCH", "ch", owner, errorMap, "");
```

Comment:
Can we replace with some constant in PSConstants that is of same text as "ESTATE_BRANCH"?